docs_src/getting-started/configuration.toml

```diff
@@ -1,68 +1,72 @@
+[Writable]
+LogLevel = 'INFO'
+
 [Service]
-Host = "localhost"
-Port = 49992
-ConnectRetries = 3
+BootTimeout = 30000
+CheckInterval = '10s'
+Host = 'localhost'
```

Comment:
If the user is using container to run other EdgeX micro services, it should be the IP of the system host in the docker network.